### PR TITLE
added favicon, opengraph, changed wording

### PR DESCRIPTION
**Pull request by ror-shubham**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,7 +5,19 @@
...
+    <title>æternity explorer</title>
+
+    <meta property="og:title" content="æternity explorer">
+    <meta property="og:site_name" content="æternity explorer">
+    <meta property="og:url" content="https://explorer.aepps.com">
+    <meta property="og:description" content="æternity blockchain explorer shows detailed information on blocks and transactions.">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="/explorer-open-graph.png">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= BASE_URL %>apple-touch-icon.png">
```
Comment (sadiqevani) [inline public/index.html]: @ror-shubham Previously when I did this for token-migration, I had some issues with the BASE_URL and images being displayed when it was shared on Facebook or other social media. I'll approve this but can you take a look if it works correctly?

File: public/index.html [hunk 2]
```diff
@@ -5,7 +5,19 @@
...
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,500|Roboto:400,500" rel="stylesheet">
-    <title>Blockchain Explorer</title>
+    <title>æternity explorer</title>
+
+    <meta property="og:title" content="æternity explorer">
+    <meta property="og:site_name" content="æternity explorer">
+    <meta property="og:url" content="https://explorer.aepps.com">
+    <meta property="og:description" content="æternity blockchain explorer shows detailed information on blocks and transactions.">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="/explorer-open-graph.png">
```
Comment (sadiqevani) [inline public/index.html]: @ror-shubham same thing here for the image, it needs the `BASE_URL` but also check if the `BASE_URL` is the correct one that points to the assets.

**Comment by ror-shubham** [comment]
@sadiqevani Yes, it doesn't work with Facebook :disappointed:

**Comment by ror-shubham** [comment]
@sadiqevani Please review again. I have changed the image:url to explorer.aepps.com/... which doesn't exist yet but explorer gets deployed after merging.

**Comment by sadiqevani** [comment]
@ror-shubham I saw you removed `og:url` does it still work without it?

**Comment by ror-shubham** [comment]
@sadiqevani yes it does

**Pull request merged**